iOS/ViewControllers/Account/AppearanceSettingsViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AppearanceSettingsViewController: UITableViewController {
+
+    @available(iOS 13.0, *)
+    private var theme: Theme {
+        get {
+            return UserDefaults.standard.theme
+        }
+        set {
+            UserDefaults.standard.theme = newValue
+            self.configureStyle(for: newValue)
+        }
+    }
+
+    @available(iOS 13.0, *)
+    private func configureStyle(for theme: Theme) {
+        UIApplication.shared.windows.forEach { window in
+            window.overrideUserInterfaceStyle = theme.userInterfaceStyle
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 3
```

Comment:
After adopting the `CaseIterable` protocol for `Theme`, you can use `Theme.allCases.count` to avoid this magic number.